sp-palt04

tpalus opened the pull request:
Posielam pilotnú verziu SP 
ešte nieje úplne hotová, chýby mi tam do implementovať akcie, ktoré može vykonávat admin (vytváranie, editovanie, mazanie produktu)
triedenie produktov podľa categorii, zobrazovanie produktov v slidery

[https://eso.vse.cz/~palt04/sp-palt04](https://eso.vse.cz/~palt04/sp-palt04)

tpalus committed:
palt04-sp

tpalus committed:
aktualizacia sp-palt04

tpalus committed:
Delete functions.php

tpalus committed:
aktualizace sp-palt04

tpalus committed:
Delete categoryView.php

tpalus committed:
Delete registration.php

tpalus committed:
Delete orders.php

tpalus committed:
Delete order-detail.php

tpalus committed:
Delete sliderView.php

tpalus committed:
Delete productView.php

tpalus committed:
Delete users.php

tpalus committed:
Delete remove-item-from-cart.php

tpalus committed:
Delete update-product.php

nvbach91 reviewed (changes requested):
- chyba v navigaci: 
![image](https://user-images.githubusercontent.com/20724910/122434784-b0505c80-cf97-11eb-975f-c9296cac3ed4.png)
- opravit
![image](https://user-images.githubusercontent.com/20724910/122434895-cc53fe00-cf97-11eb-8a2f-4cdd9663fd09.png)
- na strankach s produkty kde mate jenom jedno strankovani, to strankovani nezobrazujte
- proc kdyz si neco koupim tak se mi to vrati zpatky na index.php?
- odkaz na kosik dejte na hlavni navigaci a k tomu zobrazujte pocet aktivnich polozek v kosiku
- do polozek v kosiku pridejte odkaz na produkt
- naformatujte ceny
- pridejte moznost volby dopravy a zpusobu uhrady a evidujte je v objednavce
- proc nevidim nic v order-detail?

tpalus committed:
update prace

zobrazovanie aktualny pocet veci v kosiku, filtrovanie ovjednavok od najnovsej, oop pre pouzivatelou, detail objednavky
